packages/next-codemod/bin/upgrade.ts

```diff
@@ -178,11 +178,15 @@ async function suggestTurbopack(packageJson: any): Promise<void> {
     return
   }
 
+  console.log(
+    `${chalk.yellow('⚠')} Could not find "${chalk.bold('next dev')}" in your dev script.`
+  )
+
   const responseCustomDevScript = await prompts(
     {
       type: 'text',
       name: 'customDevScript',
-      message: 'Please add `--turbo` to your dev command:',
+      message: 'Please add "--turbo" to your dev command manually.',
```

Comment:
what is this prompting? Seems like it should just be part of the above warning but maybe I'm misunderstanding the prompt